### DISCUSSION TITLE
Running the bot on a Raspberry 4? -SOLVED-

### DISCUSSION THREAD

**Discussion by polykoma** [Zero6992/chatGPT-discord-bot#87] [General]
Hey Zero,

first of all-thank you for the amazing work!

I wondered if the bot would run succesfully on a raspberry pi 4 (4GB Ram)?

I tried to install it and that worked out of the box. 
But running the script failed. 
I have Python 3.9 installed  as well as discord.py already since I run another bot with that.

The output I get is when I run it after closely doing all the readme.md steps:

"libcblas.so.3: cannot open shared object file: No such file or directory"

My python is Python3.9 in usr/bin/python3
NumpyVersion 1.24.0

The folder with your latest github sits on desktop.

Would sincerely appreaciate your input or a tailored guide for raspberry users.
I know thats probably a lot to ask for :/

Thank you so much 
BR
poly

**Comment by Zero6992** [comment]
Hi, I'm not very familiar with Raspberry Pi-related things

I looked up your error message, have you tried installing these dependencies? 
https://stackoverflow.com/questions/53347759/importerror-libcblas-so-3-cannot-open-shared-object-file-no-such-file-or-dire

Sorry for the late reply, it's finals week at school

**Comment by polykoma** [comment]
Hey dear,

Ohhh wow. It must have been late yesterday since I couldnt figure it on my own. Im so sorry to have bothered you with such a simple questions.  
I followed the link and installed the dependencies and it worked instantly.

Means bot runs successfull on a Pi4 <3

Thank you so much 

Have a blessed Christmas Time

BR
